packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -449,210 +528,170 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
                 if (remainingHbarsBefore > maxSpendingLimit) {
-                  expect(amountSpent + estimatedTxFee).to.be.gt(maxSpendingLimit);
+                  expect(amountSpent + expectedTxCost).to.be.gt(maxSpendingLimit);
                 } else {
-                  expect(remainingHbarsAfter).to.be.lt(estimatedTxFee);
+                  expect(remainingHbarsAfter).to.be.lt(expectedTxCost);
                 }
+
+                // should allow another same tiered user to make calls
+                const differentAccount = (await createAliasAndAssociateSpendingPlan(subscriptionTier)).aliasAccount;
+                await expect(deployContract(largeContractJson, differentAccount.wallet)).to.be.fulfilled;
               }
             });
           };
 
           describe('given a valid JSON file with pre-configured spending plans', async () => {
             const SPENDING_PLANS_CONFIG_FILE = ConfigService.get('HBAR_SPENDING_PLANS_CONFIG_FILE') as string;
             const configPath = findConfig(SPENDING_PLANS_CONFIG_FILE);
-            const rawData = fs.readFileSync(configPath!, 'utf-8');
-            const expectedNonBasicPlans2 = JSON.parse(rawData) as SpendingPlanConfig[];
-
-            it('Should successfully populate all pre-configured spending plans', async () => {
-              expectedNonBasicPlans2.forEach(async (expectedPlan) => {
-                const hbarSpendingPlan = await hbarSpendingPlanRepository.findByIdWithDetails(
-                  expectedPlan.id,
-                  requestDetails,
-                );
-                expect(hbarSpendingPlan.active).to.be.true;
-                expect(hbarSpendingPlan.id).to.eq(expectedPlan.id);
-                expect(hbarSpendingPlan.subscriptionTier).to.eq(expectedPlan.subscriptionTier);
-
-                if (expectedPlan.ethAddresses) {
-                  expectedPlan.ethAddresses.forEach(async (evmAddress) => {
-                    const associatedPlanByEVMAddress = await ethAddressSpendingPlanRepository.findByAddress(
-                      evmAddress,
-                      requestDetails,
-                    );
-                    expect(associatedPlanByEVMAddress.planId).to.eq(expectedPlan.id);
-                    expect(associatedPlanByEVMAddress.ethAddress).to.eq(evmAddress);
-                  });
-                }
 
-                if (expectedPlan.ipAddresses) {
-                  expectedPlan.ipAddresses.forEach(async (ipAddress) => {
-                    const associatedPlanByIpAddress = await ipSpendingPlanRepository.findByAddress(
-                      ipAddress,
-                      requestDetails,
-                    );
-                    expect(associatedPlanByIpAddress.planId).to.eq(expectedPlan.id);
-                    expect(associatedPlanByIpAddress.ipAddress).to.eq(ipAddress);
-                  });
-                }
+            if (configPath) {
```

Comment:
**Note for Reviewers:** 
Just simply add a check to run this test suite only if configPath is not null